File: akka-cookbook/Chapter01/CustomMailbox/src/main/scala/MyActor.scala
package com.packt.chapter01

import akka.actor.{Actor, ActorRef}
import akka.actor.AbstractActor.Receive

class MyActor extends Actor {
  override def receive: Receive = {
    case (msg: String, actorRef: ActorRef) => actorRef ! msg
    case msg => println(msg)
  }
}

File: akka-essentials/Chapter05/MyRouter/src/main/java/MyActor.java
package akka.examples;

import akka.actor.AbstractActor;

public class MyActor extends AbstractActor {
    
    public Receive createReceive() {
        return receiveBuilder()
            .matchAny(message -> {
                System.out.println("MyActor: message=" + message);
            })
            .build();
    }

}